### PR TITLE
AVRO-3690: [lang/java/avro] Fix flaky test ‘testMultipleFieldAliases’

### PR DESCRIPTION
**Pull request by Jinzhengxu**
## What is the purpose of the change

This pull request fixes the flaky test 'testAnnotationMultiAvroMeta', which was causing issues with [AVRO-3690](https://issues.apache.org/jira/browse/AVRO-3690).

## Verifying this change

Run this maven command to reproduce the issue:
```
mvn -pl lang/java/avro  edu.illinois:nondex-maven-plugin:2.1.1:nondex -Dtest=org.apache.avro.reflect.TestReflect#testMultipleFieldAliases
```

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Review by clesaec — APPROVED** [review]
LGTM

File: lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java [hunk 1]
```diff
@@ -1274,12 +1274,14 @@ public void namespaceDefinitionOnFieldAliasMustThrowException() {
...
-
-    Schema expectedSchema = SchemaBuilder.record(ClassWithMultipleAliasesOnField.class.getSimpleName())
-        .namespace("org.apache.avro.reflect.TestReflect").fields().name("primitiveField").aliases("alias1", "alias2")
-        .type(Schema.create(org.apache.avro.Schema.Type.INT)).noDefault().endRecord();
-
-    check(ClassWithMultipleAliasesOnField.class, expectedSchema.toString());
+    Field field = new Field("primitiveField", Schema.create(Schema.Type.INT));
+    field.addAlias("alias1");
+    field.addAlias("alias2");
+    Schema avroMultiMeta = Schema.createRecord("ClassWithAliasOnField", null, "org.apache.avro.reflect.TestReflect",
```
Comment (martin-g) [inline lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java:1280]:
```suggestion
    Schema avroMultiMeta = Schema.createRecord("ClassWithMultipleAliasesOnField", null, "org.apache.avro.reflect.TestReflect",
```

**Review by martin-g — APPROVED** [review]

File: lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java (hunk 1 above)
Comment (Jinzhengxu) [reply lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java]: Thank you for correcting me.

**Review by clesaec — APPROVED** [review]
LGTM

**Comment by nielsbasjes** [comment]
@Jinzhengxu Can you please rebase this? Thanks.

**Comment by Jinzhengxu** [comment]
@nielsbasjes Sure. I will rebase this ASAP.

**Comment by nielsbasjes** [comment]
Thanks!

**Pull request merged**